stm32mp1.mk

```diff
@@ -59,6 +59,7 @@ LINUX_DTB_BIN		:= $(STM32MP1_DTS_BASENAME).dtb
 BINARIES_PATH		?= $(ROOT)/out/bin
 TFA_PATH		?= $(ROOT)/trusted-firmware-a
 U_BOOT_PATH		?= $(ROOT)/u-boot
+SCP_FWM_PATH		?= $(ROOT)/scp-firmware
```

Comment:
FWM is a strange abbreviation for FirMWare, how about FW?

Reply:
> FWM is a strange abbreviation for FirMWare, how about FW?

My fault. FMW or FW would be better.
I'll change to `SCPFW_PATH`, to match the `CFG_SCPFW_*` switches we have in optee_os.